### DISCUSSION TITLE
2022-03: Goodbye, Google Workspace

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#323] [Announcements]
# 2022-03: Goodbye, Google Workspace

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-03/

**Comment by axionl** [comment]
在你跑到 fastmail 的时候我刚好跑去自建 🤣

**Reply by Xuanwo** [reply]
勇啊

**Comment by 5uy4n9** [comment]
> 还能增加任意的 alias

看官方文档，应该是600个 
https://www.fastmail.help/hc/en-us/articles/1500000277382-Account-limits

**Reply by Xuanwo** [reply]
感谢指正！
